package.json

```diff
@@ -54,25 +54,25 @@
     "test:e2e:remote:run": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' -- npx playwright test tests/playwright/src/special-specs/podman-remote/ -g @podman-remote",
     "test:e2e:update": "pnpm run test:e2e:build && pnpm run test:e2e:update:run",
     "test:e2e:update:run": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' -- npx playwright test tests/playwright/src/special-specs/installation/ -g @update-install",
-    "test:main": "vitest run -r packages/main --passWithNoTests --coverage",
-    "test:preload": "vitest run -r packages/preload --passWithNoTests --coverage",
-    "test:preload-docker-extension": "vitest run -r packages/preload-docker-extension --passWithNoTests --coverage",
-    "test:preload-webview": "vitest run -r packages/preload-webview --coverage",
-    "test:extensions": "vitest run -r extensions --passWithNoTests --coverage && pnpm run test:extensions:compose && pnpm run test:extensions:kind && pnpm run test:extensions:docker && pnpm run test:extensions:lima && pnpm run test:extensions:kube && pnpm run test:extensions:podman && pnpm run test:extensions:registries && pnpm run test:extensions:kubectl-cli",
```

Comment:
wondering if it should be:
`pnpm -r --filter './extensions/**' test`

so it's in //

Reply:
not sure to understand the synthax

Reply:
it's to run the command `test` in all the folders matching '**/extensions/*'

Reply:
(idea behind would be to find a way to reduce the entries in the package.json)
where for example we could run `pnpm test:extension <extension-name>`

Reply:
Maybe we could make the `--project=extensions/*` working (I don't if they support it, but we could add it anyway)